### PR TITLE
Arabica Overlay

### PR DESCRIPTION
**Pull request by TheWayYouLOOKATME**
### `namespace ArabicaCliento.Commands;`

🕵️‍♂️ **arabica.overlay**

Shows CKEYs and CharNames of players in the range of the local player.

📜 **arabica.entities**

Writes in console coordinates, CharNames, Ckeys, Overall health and world map coordinates of entity, each of them controlled by player.

## Summary by Sourcery

Add a new overlay feature to display player information within range and implement console commands for entity listing and overlay toggling.

New Features:
- Introduce an overlay feature that displays CKEYs and character names of players within the local player's range.
- Add a console command 'arabica.entities' to output a list of entities controlled by players, including their coordinates, character names, CKEYs, and health status.

Enhancements:
- Implement a toggle command 'arabica.overlay' to enable or disable the overlay feature.

### CODE REVIEW

**Comment by voko421** [comment]
Селфмёрдж жди докс

**Comment by noverd** [comment]
Он мейнтейнер, ему можно

**Comment by noverd** [comment]
вернее сказать коовенер

File: ArabicaCliento/Commands/Arabica.EntityList.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+{
+    public string Command => "arabica.entities";
+    public string Description => "Outputs a entity(human-controlable) list with their id, name and coordinates";
+    public string Help => "arabica.entities";
+
+    public void Execute(IConsoleShell shell, string argStr, string[] args)
+    {
+        var entityManager = IoCManager.Resolve<IEntityManager>();
+
+        foreach (var entity in entityManager.GetEntities())
```
Comment (misandrie) [inline ArabicaCliento/Commands/Arabica.EntityList.cs:25]: Автор ты что сука наделал

Comment (TheWayYouLOOKATME) [reply ArabicaCliento/Commands/Arabica.EntityList.cs:25]: мне такое можно, ни .гг а

Comment (misandrie) [reply ArabicaCliento/Commands/Arabica.EntityList.cs:25]: ты букву "я" пропустил

Comment (voko421) [reply ArabicaCliento/Commands/Arabica.EntityList.cs:25]: > да, извини, ошибся, попу тал, я ни г. Ер

Кролик двойная банда

Comment (noverd) [reply ArabicaCliento/Commands/Arabica.EntityList.cs:25]: github ban evasion

**Pull request merged**